x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/render_as_select/render_as_select.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiButtonGroup } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ES_GEO_FIELD_TYPE, RENDER_AS } from '../../../../../common/constants';
+import { getIsCloud } from '../../../../kibana_services';
+import { getIsGoldPlus } from '../../../../licensed_features';
+import { CLUSTER_LABEL, GRID_LABEL, HEX_LABEL } from './i18n_constants';
+import { ShowAsLabel } from './show_as_label';
+
+interface Props {
+  geoFieldType?: ES_GEO_FIELD_TYPE;
+  renderAs: RENDER_AS;
+  onChange: (newValue: RENDER_AS) => void;
+  isColumnCompressed?: boolean;
+}
+
+export function RenderAsSelect(props: Props) {
+  if (props.renderAs === RENDER_AS.HEATMAP) {
+    return null;
+  }
+
+  let isHexDisabled = false;
+  let hexDisabledReason = '';
+  if (!getIsCloud() && !getIsGoldPlus()) {
+    isHexDisabled = true;
+    hexDisabledReason = i18n.translate('xpack.maps.hexbin.license.disabledReason', {
+      defaultMessage: '{hexLabel} is a subscription feature.',
```

Comment:
This should be:

xxx is a [subscription feature](https://www.elastic.co/subscriptions).

Reply:
Should links be in tooltips? There is no way to click the link.

Reply:
Correct.  Do you have a quick screenshot of where this copy is? Can see if there are maybe some other options.

Reply:
<img width="300" alt="Screen Shot 2022-03-18 at 3 54 23 PM" src="https://user-images.githubusercontent.com/373691/159088914-368a9698-d2f2-4dc2-8a02-f9cfa02ed803.png">